composer.json

```diff
@@ -28,18 +28,18 @@
     }
   ],
   "require": {
-    "php": ">=7.0.8",
     "ckeditor/panelbutton": "^4.10",
+    "composer/installers": "^1.2",
     "cweagans/composer-patches": "^1.6.5",
-    "drupal-composer/drupal-scaffold": "^2.5",
+    "drupal/core-composer-scaffold": "^9",
     "drupal/better_normalizers": "^1.0@beta",
     "drupal/block_content_permissions": "^1.8",
-    "drupal/components": "^1",
-    "drupal/config_installer": "^1",
-    "drupal/core-recommended": "^8.8.0",
+    "drupal/components": "^2",
+    "drupal/core-recommended": "^9.0",
+    "drupal/core-vendor-hardening": "^8.8",
```

Comment:
Why not not use `^9.0` for all core provided components?

Reply:
Changed all core packages to `^9.0` as suggested